catalyst/data/sampler.py

```diff
@@ -154,4 +159,22 @@ def __len__(self) -> int:
         return self.mini_epoch_len
 
 
-__all__ = ["BalanceClassSampler", "MiniEpochSampler"]
+class DistributedSamplerOverSampler(DistributedSampler):
```

Comment:
btw, what do you think about `DistributedSamplerWrapper` naming?

Reply:
done